src/components/utils/error/ContactSupport.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * A component that can be used to allow a user to contact support.
+ */
```

Comment:
I also extracted this from `ErrorHandler.js`. Once again, this seemed to me to be the most reasonable thing to do, but I'm curious about what others think.